makelib/k8s_tools.mk

```
@@ -40,6 +40,10 @@ KUSTOMIZE := $(TOOLS_HOST_DIR)/kustomize-$(KUSTOMIZE_VERSION)
 OLMBUNDLE_VERSION ?= v0.4.0
 OLMBUNDLE := $(TOOLS_HOST_DIR)/olm-bundle-$(OLMBUNDLE_VERSION)
 
+# the version of up to use
+UP_VERSION ?= v0.4.0
```

```suggestion
UP_VERSION ?= v0.10.0
```
Given we're looking for features in `v0.10.0`, seems like this is a good default version?

---

Ah yes missed that in the update -- thank you!

---

@tnthornton updated 👍🏻